js/adapt-homeButton.js

```diff
@@ -42,12 +42,24 @@ define([
 
         _enabled: function() {
             this._$html.toggleClass("hide-home-button", !!this._config._hideHomeButton);
+            if (!$(".navigation-home-button")[0]) {
+                // if home button doesn't exist create home button
+                this._createHomeButton();
+            }
             if (this._config._redirectToId) {
                 this._dataEvent = $(".navigation-home-button").attr("data-event");
                 $(".navigation-home-button").attr("data-event", "redirectedHomeButton");
             }
         },
 
+        _createHomeButton: function() {
+            var $homeButton = $('<button></button>')
+                .attr('data-event', 'homeButton')
+                .addClass('base navigation-home-button icon icon-home');
+
+                $('.navigation-inner').append($homeButton);
```

Comment:
Could use the jQuery object syntax if you wanted to create without the extra methods:
```js
$('.navigation-inner').append($('<button>', {
    attr: { 'data-event': 'homeButton' },
    'class': 'base navigation-home-button icon icon-home'
}));
```